ACP pre-launch review (#1)
lsst/rtn-073: index.rst

   Data must be embargoed after arriving at the USDF - for 30 days in commissioning and 80 hrs in Survey Operations. Alert Processing will not record vetoed streaks. What rules will we impose on accessing data during that period?

Embargo Restrictions
====================

Embargoed data types cannot leave the USDF during the embargo period.

> LSST Users, such as those who have been granted USDF accounts via the Resource Allocation Process, will not have access to the data in the embargo rack.

Author: New sentence, following a prompt from @leannep, and confirmation from @richardxdubois that LSST User permissions are not specified anywhere else.

Reviewer: Suggest -- "LSST Users, including those who have been granted USDF accounts via the Resource Allocation Process, will not have access to the data in the embargo rack."

i.e. there is no path ever to a "User" getting access to embargo

Author: Good - implemented!

> LSST Users, such as those who have been granted USDF accounts via the Resource Allocation Process, will not have access to the data in the embargo rack.

Author: ```suggestion
LSST Users, including those who have been granted USDF accounts via the Resource Allocation Process, will not have access to the data in the embargo rack.
```

Author: Useful tweak suggested offline by @leannep

> Rubin Observatory staff (including the Commissioning Team) will have full access to the data in the embargo rack as their Rubin roles require. 

Author: "As their Rubin roles require" is a generalization inspired by previous comments from @leannep - no need to single out any particular Rubin roles here, it's just the case that not everyone on the Observatory staff will need access to the embargo rack. FAO @womullan and @richardxdubois

Reviewer: Might be useful to specify what the process is or who the authorizing body is that decides if embargo rack access is required.

Author: Yes - this needs discussion (and may need a separate PR).

Embargo Restrictions:
=====================

Embargoed data types cannot leave the USDF during the embargo period. 

> All Rubin USDF account holders will have full access to data in the embargo rack (unless their Rubin role requires them to only hold standard LSST User permissions, as may be the case for some Rubin Community Scientists, for example). 

Reviewer 2: If @leannep agrees ok

Reviewer: What are "standard LSST User permissions," as opposed to the permissions that  "All Rubin USDF account holders will have"  -- I do not see these terms defined in the document. Could a clear definition be added or referenced that clarifies exactly what these permissions are and the consequences.

Reviewer: Why would "some community scientists" have reduced/restricted  access to data? Why only community scientists? All current  community scientists are fully on-project and have a USDF account,  and so have the same rights as all  Rubin staff

Author: @leannep We can generalize this sentence to remove mention of Community Scientists if you prefer - I'll suggest a change. I just thought this role would be a good example for a Rubin staff member who didn't _need_ to have access to the embargo rack to do their work, and fact might even be better placed _outside the embargo rack_ with permissions the same as the LSST Users so that they can appreciate the challenges faced by those community members.  But there are other Rubin roles that will not require the staff in them to be able to access to the embargo rack - DRP pipeline scientists, DRP systematics scientists, etc etc.

Author: @leannep Regarding "standard LSST User permissions". I assume that when we have LSST Users at the USDF, because they have successfully proposed for resources there through the RAC, they will have permissions set such that they cannot access the embargo rack. By definition, the data in the embargo rack has not yet been released to the LSST Users, so they will have no need (or right) to access the embargo rack. If check whether RDM has any documentation yet on LSST User accounts at the USDF - if it doesn't, I'll reword to avoid using terminology that doesn't exist yet...

Removal from the embargo repo can occur after a successful copy to normal storage, or it can be postponed for a configurable time period, but space limitations mean that no dataset should be expected to be retained for more than 30 days in the embargo repo.

> Handling of the Satellite Catalog

Reviewer 2: good !

=================================

> The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. The catalogs may not leave that directory, except for transient copies needed by prompt/alert processing. No derivative catalogs may be stored.

Reviewer 2: I worry this prevents us coming up with a temporary catalog and keeping it on embargo .. I think you mean no derivative maybe stored outside embargo rack.

Reviewer 3: Not from my reading. They say we have to treat any such catalogue as OUO... we're trying to avoid that...

Author: Correct. I'm investigating whether tables of performance metrics count as "derivative catalogs" - I don't think they should.

Author: Let's keep the line about no derivative catalogs. John Sarrao confirmed that tabulating metrics is OK, and this is what @ebellm needs most, IIUC.

Removal from the embargo repo can occur after a successful copy to normal storage, or it can be postponed for a configurable time period, but space limitations mean that no dataset should be expected to be retained for more than 30 days in the embargo repo.

Handling of the Satellite Catalog
=================================

> The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 

Author: ```suggestion
The catalog will be protected as if it was CUI Basic (as defined in [DOE Order 471.7](https://directives.dev.doxcelerate.com/directives-documents/400-series/0471.7-BOrder)). As such, access will be restricted to authorized individuals, identified in an approved access list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 
```

Author: Spotted this, @richardxdubois : since the access policy only states that the catalog "is for government official use only", and the catalog itself is not marked as CUI, I think we conclude that the catalog is potentially OUO and hence is "legacy matter", such that we need to follow Section 4.f.8 of order 471.7 (_"Legacy documents and matter that will retain Official Use Only (OUO) markings or have not been reviewed to determine if they contain CUI should also be safeguarded in accordance with the requirements established in this Directive. If safeguarding requirements cannot be met, alternate plans for appropriately protecting documents and matter that may potentially contain CUI must be developed. This alternate protection plan must be approved by the Departmental Element Designated CUI Official
or delegate."_) I'm using "protected as if it was" as shorthand for "safeguarded in accordance with" but we could use the order's wording directly.

Reviewer 2: You could just state ts protected with out saying anything about CUI. Ed really did to want us putting the word CUI anywhere even to say it NOT CUI...

Author: OK - I like your suggestion below. We can show that we know what we are doing re: OUO/CUI handling in the ACP, which is not public.

> The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 

Reviewer 2: ```suggestion
The catalog, as per DOE Order 471.7  will be restricted to authorized individuals, identified in an approved access list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 
```
